### ISSUE TITLE
wrong info under plugin-info

### ISSUE THREAD

**Issue by olape-git** [cmsimple-xh/expandcontract#10]
![image](https://user-images.githubusercontent.com/35469568/161609602-f51ae815-63be-44b5-9f0a-dfd0ba021db3.png)

**Comment by frase-git** [comment]
Was ist daran falsch?

**Comment by olape-git** [comment]
`{{{expand 'Seite 1, Seite 2, Seite 3, Seite 4'}}}` --> `{{{expand('pages = Seite 1, Seite 2, Seite 3, Seite 4')}}}`
`{{{expand 'Seite X', 'Linktext'}}}` --> `{{{expand('pages = Seite X', 'headlines = Linktext')}}}`

**Comment by frase-git** [comment]
Danke. Klassischer Fall von Betriebsblindheit.

**Comment by frase-git** [comment]
Übrigens: Dein Code hat auch ein paar Klammern **zu viel**.
Die Parameter müssen hier (eigenartigerweise) keine Klammern ( ) haben.
Oder, bin ich da schon wieder auf dem Holzweg?

**Comment by TN03** [comment]
Irgendwo stand auch, dass bei Angabe eines Seitennamens die "erste gefundene" Seite genommen wird. Das stimmt ja. Normalerweise müssten Seiten mit gleichem Namen in unterschiedlichen Ebenen, bzw. unter unterschiedlichen Elternseiten trotzdem ansprechbar sein, indem man den Pfad entsprechend angibt: `Allgemeine Frage/Antwort 1`  `Andere Frage/Antwort 1`
Eventuell wäre ein Hinweis darauf ganz gut.

**Comment by frase-git** [comment]
>  Eventuell wäre ein Hinweis darauf ganz gut.

Im Moment funktioniert das mit den Pfaden ja nicht - oder?
Also ist doch der Hinweis ausreichend, dass immer nur die erste im Seitenbaum gefundene Seite genommen wird.

**Comment by TN03** [comment]
> Im Moment funktioniert das mit den Pfaden ja nicht - oder?

Das müsste doch gehen: `'pages = Eine Seite/Seite1, Andere Seite/Seite 1'` oder irre ich mich da???

**Comment by olape-git** [comment]
> Übrigens: Dein Code hat auch ein paar Klammern **zu viel**. Die Parameter müssen hier (eigenartigerweise) keine Klammern ( ) haben. Oder, bin ich da schon wieder auf dem Holzweg?

Ich schreibe es immer mit ().


> Das müsste doch gehen: `'pages = Eine Seite/Seite1, Andere Seite/Seite 1'` oder irre ich mich da???

Das glaube ich eigentlich nicht. Dann hätten wir pages mit $u umsetzen müssen, nicht mit $h.
Ich habe es nicht getestet, aber es sollte mich wundern.

**Comment by TN03** [comment]
> > Das müsste doch gehen: `'pages = Eine Seite/Seite1, Andere Seite/Seite 1'` oder irre ich mich da???
> 
> Das glaube ich eigentlich nicht. Dann hätten wir pages mit $u umsetzen müssen, nicht mit $h. Ich habe es nicht getestet, aber es sollte mich wundern.

Stimmt. Kann hier noch nicht gehen.  Verwechselt, sorry.

Ich habe bereits an einer anderen Alternativ-Version gebastelt, die dann (optional) gar keine Unterseiten braucht, sondern alles aus einer Seite holt.
Momentan "passt" das aber noch nicht richtig zusammen. Weiß auch nicht, ob ich das weiter verfolge.

**Comment by olape-git** [comment]
> sondern alles aus einer Seite holt.

Ich finde das gerade gut, dass man den Inhalt etwas aufteilen kann.
Das macht das Bearbeiten leichter / übersichtlicher.
Wenn alles in einer Seite steht ist das Bearbeiten wesentlich schwerer, auch wenn man mal im Queltext ist.

**Comment by TN03** [comment]
> Ich finde das gerade gut, dass man den Inhalt etwas aufteilen kann.

Ja, Ansichtssache. Mir sind das zu viele Seiten in der Navi. Da muss man ewig suchen, scrollen, Zweige öffnen. Eine Seite / je Akkordeon finde ich für meine (Kunden-)Projekte simpler. Gesplittet wird nach (konfigurierbarer) Überschrift. 
Und ohne Edit / Preview - Links sind viele mit der jetzigen Lösung auch schon überfordert.

Quelltext? Das klappt bei meinen Durchnittsusern in der Regel so oder so nicht. 

Und was die Übersicht angeht: alles in einer Seite macht es viel übersichtlicher und einfacher in der Bearbeitung.

BTW: Aktuell habe ich mal versucht "Snippets" (wie von Frank) bei einem Projekt zu vermitteln. Den Block am richtigen Platz im Quelltext zu positionieren funktioniert schon gut. Aber nur, so lange ich gerade erkläre und vorführe. Einen Tag später war alles vermurkst...

Aber das muss ja hier in diesem Plugin nicht so sein und aktuell brauche ich das auch nicht.

**Comment by olape-git** [comment]
> Ja, Ansichtssache. Mir sind das zu viele Seiten in der Navi. Da muss man ewig suchen, scrollen, Zweige öffnen. Eine Seite / je Akkordeon finde ich für meine (Kunden-)Projekte simpler. Gesplittet wird nach (konfigurierbarer) Überschrift.
> Und ohne Edit / Preview - Links sind viele mit der jetzigen Lösung auch schon überfordert.

Hm, ich pflege aktuell eine Site, da ist so etwas über Accordions (JS / CSS) realisiert.
Da ist der normale Seiteninhalt und dann der Inhalt der Accordions eben alles in einem.
Das ist nicht wirklich lustig.

`{{{expand('one-page = Seite X')}}}` --> "Gesplittet wird nach (konfigurierbarer) Überschrift." --> und alles andere könnte bleiben wie gehabt.

**Comment by TN03** [comment]
> Da ist der normale Seiteninhalt und dann der Inhalt der Accordions eben alles in einem.
> Das ist nicht wirklich lustig.

Das geht natürlich nicht, klar. Aber eine Seite / Akkordeon finde ich gut.


> `{{{expand('one-page = Seite X')}}}` --> "Gesplittet wird nach (konfigurierbarer) Überschrift." --> und alles andere könnte bleiben wie gehabt.

Ist schon versuchsweise in der Mache :). 
Allerdings stehen jetzt erstmal dringend andere Sachen auf dem Plan.
